velox/functions/sparksql/tests/DateTimeFunctionsTest.cpp

```diff
@@ -757,5 +758,118 @@ TEST_F(DateTimeFunctionsTest, fromUnixtime) {
       fromUnixTime(0, "yyyy-MM-dd HH:II"), "Specifier I is not supported");
 }
 
+TEST_F(DateTimeFunctionsTest, makeTimestamp) {
+  const auto testMakeTimestamp = [&](const RowVectorPtr& data,
+                                     const VectorPtr& expected,
+                                     bool hasTimeZone) {
+    auto result = hasTimeZone
+        ? evaluate("make_timestamp(c0, c1, c2, c3, c4, c5, c6)", data)
+        : evaluate("make_timestamp(c0, c1, c2, c3, c4, c5)", data);
+    facebook::velox::test::assertEqualVectors(expected, result);
```

Comment:
Maybe we need to test different encodings as a fast path for constant encoding is implemented. `testEncodings` could be used for that purpose.

Reply:
@rui-mo Rui has a good point. Would you address her comment?

Reply:
@mbasmanova Method `testConstantTimezone` below was used to address this comment.